blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -31,8 +31,9 @@ const QueuedState = () => (
                 size: 20,
                 icon: 'timer',
                 style: { fill: '#fff' },
-            }} />
-            <span className="waiting">Waiting for run to start.</span>
```

Comment:
Very strange, that must have been a mismerge. Definitely didn't delete it intentionally. Will fix